flexmeasures/cli/data_add.py

```diff
@@ -922,7 +922,8 @@ def create_schedule(
             soc_min=soc_min,
             soc_max=soc_max,
             roundtrip_efficiency=roundtrip_efficiency,
-            price_sensor=optimization_context_sensor,
+            up_deviation_price_sensor=optimization_context_sensor,
```

Comment:
Shall we then revert to using "price" instead of "optimization context", and use "consumption/production price sensor" throughout? Then we'd write in the documentation that this can also be used to point to a sensor that records emission intensity, or any other thing per unit of energy, that one would want to minimize.

Reply:
I implemented my own suggestion, with a future warning preparing existing users for deprecation of the `optimization-context-id` CLI option.